SquareLine/gen.py

```diff
@@ -0,0 +1,231 @@
+from genericpath import isdir
+import sys
+import os
+import json
+import shutil
+from colorama import Fore
```

Comment:
This is not a built-in Python module. You could mention in the readme how to install dependencies and here, catch the ImportError and print a nice error message with suggestion to solve it.